vendor/k8s.io/utils/exec/testing/hello

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
@dims - non-blocking for this PR, but might be worth a cleanup PR in utils to move this into a `testdata` folder so it doesn't get vendored unnecessarily

Reply:
Nice suggestion, I will take it.

Reply:
👍🏾

Reply:
Ref https://github.com/kubernetes/utils/pull/252